src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -247,6 +249,67 @@ void ManageSurfaceHeatBalance(EnergyPlusData &state)
 // Beginning Initialization Section of the Module
 //******************************************************************************
 
+Real64 truncateOverrideValue(Real64 value)
```

Comment:
Breaking these things out into functions is not actually helping, it's just making you pull out and pass parameters around.

Reply:
Removed the function and inlined the content